### PR TITLE
Adding a filter for the path column in the dependencies view

### PR DESCRIPTION
**Pull request by Geni2**
I introduced the changes almost 6 months ago for adding a filter into the dependencies view for the path column and now I'm facing some issues there because the AdminBundle was deprecated from the Pimcore project I'm doing the relevant changes here according to the documentation and the other changes will be in my Pull Request that is open in Pimcore which I will edit and remove `dependencies.js` and `ElementController.php` file from there.

Here is the link to my other Pull Request: https://github.com/pimcore/pimcore/pull/14671

### CODE REVIEW

**Comment by Geni2** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Geni2** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Geni2** [comment]
rechek

**Comment by Geni2** [comment]
recheck

**Comment by dvesh3** [comment]
@Geni2 please see the tests are failing, which seems related to the changes in the PR. thanks!

**Comment by Geni2** [comment]
> @Geni2 please see the tests are failing, which seems related to the changes in the PR. thanks!

@dvesh3 I'm working to fix those failing tests asap. Thank you

File: public/js/pimcore/element/dependencies.js [hunk 1]
```diff
@@ -110,8 +110,10 @@ pimcore.element.dependencies = Class.create({
             model: requiresModel
         });
 
+        var plugins = ['pimcore.gridfilters'];
```
Comment (robertSt7) [inline public/js/pimcore/element/dependencies.js]:
```suggestion
        const plugins = ['pimcore.gridfilters'];
```

File: public/js/pimcore/element/dependencies.js [hunk 2]
```diff
@@ -218,8 +220,10 @@ pimcore.element.dependencies = Class.create({
 
         });
 
+        var plugins = ['pimcore.gridfilters'];
```
Comment (robertSt7) [inline public/js/pimcore/element/dependencies.js]:
```suggestion
        const plugins = ['pimcore.gridfilters'];
```

**Review by robertSt7 — CHANGES_REQUESTED** [review]

**Comment by robertSt7** [comment]
@Geni2 Any news on this?

**Comment by robertSt7** [comment]
@Geni2 Please merge 1.x into your branch. Thanks

**Comment by Geni2** [comment]
> @Geni2 Any news on this?

Working on this and I hope soon will be some updates here

**Comment by Geni2** [comment]
@robertSt7 With the commit that I just made I have included also your requested changes but I did it in a different way. Thank you

**Comment by robertSt7** [comment]
@Geni2 Thanks for your effort

**Pull request merged**